src/families/ethereum/synchronisation.ts

```diff
@@ -57,8 +53,7 @@ export const getAccountShape: GetAccountShape = async (
     initialAccount &&
     areAllOperationsLoaded(initialAccount) &&
     mostRecentStableOperation &&
-    !outdatedSyncHash &&
-    NFT_VERSION
```

Comment:
Versionning is not useful anymore because we are now regenerating all NFTs from all ops at each sync (to avoid pontetial missing txs from the explorers) instead of doing it incrementaly. 
NFT Versioning was a way to force a full resync in case we changed anything to the NFT type, or if we added new standards not indexed yet for example.